source/manual/aws-iam-key-rotation.html.md

```diff
@@ -29,57 +29,67 @@ Log in to the [IAM Management Console](https://console.aws.amazon.com/iam/home?r
 for the environment you want to rotate keys for and you'll see a list of users
 and the age of their access keys.
 
-### Find users access key
+### 1. Find users access key
 
-Click on the user who's key you want to rotate, then click on "Security Credentials"
-to see their access key.
+Click on the user whose key you want to rotate. Click on `Security Credentials`. Make a note of the Access Key ID.
 
-### 1. Find where the key is used
+### 2. Find where the key is used
 
-Make a note of the Access Key ID and find where this access key is defined in
-[govuk-secrets](https://github.com/alphagov/govuk-secrets). Keys may also be
-defined in control panels for 3rd party services such as Fastly or Logit. Keys
-may also be defined in more then one place, for example the hieradata values
+Enter [govuk-secrets/puppet](https://github.com/alphagov/govuk-secrets/puppet) in your machine.
```

Comment:
This link didn't work for me but https://github.com/alphagov/govuk-secrets/tree/master/puppet does

Reply:
Updated 👍